alerts/alerts.go

```diff
@@ -273,6 +274,20 @@ func (qp *QueueProcessor) Deliver(req *alert.AlertRequest, ctx *AlertContext) er
 // Run loops while there are any unprocessed alerts and attempts to deliver them.
 func (qp *QueueProcessor) Run(config *evergreen.Settings) error {
 	startTime := time.Now()
+	adminSettings, err := admin.GetSettingsFromDB()
+	if err != nil {
+		grip.Error(errors.Wrap(err, "error retrieving admin settings"))
+	}
+	if adminSettings != nil && adminSettings.ServiceFlags.AlertsDisabled {
+		grip.Info(message.Fields{
+			"runner":  qp.Name(),
+			"status":  "success",
```

Comment:
Is "success" really correct here? Should it be "disabled" or something else more illuminating?